### PR TITLE
[SC-721] allow returning maps from parsing functions

### PR DESCRIPTION
**Pull request by mpmiszczyk**

### CODE REVIEW

File: lib/surgex/sentry/sentry.ex [hunk 1]
```diff
@@ -5,7 +5,7 @@ defmodule Surgex.Sentry do
   **NOTE: Deprecated in favor of Elixir 1.9 runtime configuration.**
   """
 
-  alias Mix.{Config, Project}
```
Comment (psubocz) [inline lib/surgex/sentry/sentry.ex:8]: 👏

File: config/config.exs [hunk 2]
```diff
@@ -1,41 +1,3 @@
-use Mix.Config
+import Config
 
-if Mix.env() == :test do
```
Comment (psubocz) [inline config/config.exs:3]: 🙌

File: mix.exs [hunk 3]
```diff
@@ -46,7 +46,6 @@ defmodule Surgex.Mixfile do
 
   defp aliases do
     [
-      check: check_alias(),
```
Comment (psubocz) [inline mix.exs:49]: Isn't this being used, or helpful during development?

**Review by psubocz — APPROVED** [review]

File: mix.exs (hunk 3 above)
Comment (mpmiszczyk) [reply mix.exs:49]: We used to use that. It was an check on CI, but we moved onto the `fersha-elixir-orb` and now we [have job called `static-check`](https://app.circleci.com/pipelines/github/surgeventures/surgex/78/workflows/6f982968-bd84-4595-8257-2ddd18fa2c13/jobs/180) which do does those things as "normal" separate commands.

**Pull request merged**